src/index.js

```diff
@@ -0,0 +1,235 @@
+import './style.css';
+import './comment.css';
+import Logo from './dragon.jpeg';
+
+const myLogo = new Image();
+myLogo.src = Logo;
+const logoDiv = document.querySelector('#logo-div');
+logoDiv.appendChild(myLogo);
+
+const baseUri = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
+const appId = 'K5LEyqREMBDZLL96ZFuw';
+
+const addLike = async (itemId) => {
+  const response = await fetch(`${baseUri}${appId}/likes`, {
+    method: 'POST',
+    body: JSON.stringify(
+      {
+        item_id: itemId,
+      },
+    ),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  response.json().then((json) => {
+    console.log(json);
+  });
+};
+
+const getLike = (id, node) => {
+  fetch(`${baseUri}${appId}/likes`)
+    .then((response) => response.json())
+    .then((json) => {
+      let bool = false;
+      let likenum = -1;
+      json.forEach((item) => {
+        if (item.item_id === id) {
+          bool = true;
+          likenum = item.likes;
+        }
+      });
+
+      if (bool) {
+        node.textContent = `like ${likenum + 1}`;
+        addLike(id);
+      } else if (bool === false) {
+        node.textContent = 'like 1';
+        addLike(id);
+      }
+    });
+};
+
+const loadLike = (id, node) => {
+  fetch(`${baseUri}${appId}/likes`)
+    .then((response) => response.json())
+    .then((json) => {
+      let bool = false;
+      let likenum = 0;
+      json.forEach((item) => {
+        if (item.item_id === id) {
+          bool = true;
+          likenum = item.likes;
+        }
+      });
+      if (bool) {
+        node.textContent = `like ${likenum}`;
+      } else if (bool === false) {
+        node.textContent = 'like 0';
+      }
+    });
+};
+
+const loadComments = (itemId) => {
+  fetch(`${baseUri}${appId}/comments?item_id=${itemId}`)
+    .then((response) => response.json())
+    .then((json) => {
+      const itemArr = json;
+      const commentsDiv = document.querySelector('#comments');
+      let commentsHtml = '';
+      itemArr.forEach((item) => {
+        commentsHtml += `<p>${item.creation_date} by ${item.username} : ${item.comment}`;
+      });
```

Comment:
- [ ] There is an issue here when the response is not `ok`. Your function will always try to iterate through an array even if the `itemArr` is falsy. Let's fix that

![Screenshot from 2021-10-21 16-28-13](https://user-images.githubusercontent.com/40503704/138310545-a3c3e12c-70f6-438f-9d79-20e3858ee5d6.png)

![Screenshot from 2021-10-21 16-37-00](https://user-images.githubusercontent.com/40503704/138311191-e9698e35-1354-4003-84b3-cd1412dcbe63.png)

Reproduce the error:
- Click on the `Sweet and Sour Pork` comment button